pkg/cloud-provider/securitygroup/securitygroup.go

```diff
@@ -170,6 +177,18 @@ func (c *CloudResourceID) String() string {
 	return c.Name + "/" + c.Vpc
 }
 
+type CloudRuleDescription struct {
+	Name           string
+	Namespace      string
+	AppliedToGroup string
+}
+
+func (r *CloudRuleDescription) String() string {
```

Comment:
Maybe we should split all the cloud resource related definitions and funcs into a separate file, instead of reusing the securitygroup.go?

Reply:
Since description is SG rule related, I prefer to keep it here.